packages/core/src/Table/TableScrollBox.js

```diff
@@ -0,0 +1,88 @@
+import propTypes from '@dhis2/prop-types'
+import { colors } from '@dhis2/ui-constants'
+import cx from 'classnames'
+import React from 'react'
+import css from 'styled-jsx/css'
+import { Box } from '../Box/Box.js'
+
+const { className: tableClassName, styles } = css.resolve`
+    div {
+        border: 1px solid ${colors.grey300};
+        border-top: none;
+        box-sizing: border-box;
+    }
+    div > :global(table) {
+        border: none;
+    }
```

Comment:
When using a scroll box, the box should take care of the border, not the table. So again, this is violating purely scoped styles.

Reply:
Wouldn't this work the other way as well? (putting `:global(.table-scroll-box) > table` in the table styles?
Or Instead let this one render the `Table` component and accept the `Table`'s props here as well?

Reply:
> Or Instead let this one render the `Table` component and accept the `Table`'s props here as well?

This is a good idea, I like having a scrolling table. I'm not quite sure whether we should have a separate component, or simply integrate it into the existing `Table`. The current `Table `component implementation is extremely simple, so I'm not worried about this becoming too complex. I guess we can implement this by simply adding a `scrollHeight` and `scrollWidth` prop to the table.

Reply:
I think I'm fine with either way as long as we keep styles isolated / together with the component